README.md

```diff
@@ -100,6 +100,15 @@ prompt.
 To run all the examples (except those that have extra dependencies
 or require an internet connection), execute 'python run_all.py'.
 
+An alternative method is using `pip`:
+
+```
+pip install git+https://github.com/matplotlib/basemap.git
+```
+
+You migh want to run pip with administrative privilegies (e.g. `sudo pip`) or
```

Comment:
I don't think we should encourage `sudo pip`. Let's only promote the `--user` option.

Reply:
Install from the Git repositories work on MacOS Mojave with Python 3.7 and pipenv and/or pip.
Reference: https://stackoverflow.com/questions/51963619/pyproj-fails-to-compile-when-i-pip-install-it-and-its-not-about-gcc
* With ``pipenv``
```bash
$ pipenv install cython
$ pipenv install git+https://github.com/jswhit/pyproj.git#egg=pyproj
$ pipenv install git+https://github.com/matplotlib/basemap.git
```

* With just ``pip``:
```bash
$ pip install -U cython
$ pip install -U git+https://github.com/jswhit/pyproj.git#egg=pyproj
$ pip install -U git+https://github.com/matplotlib/basemap.git
```